avocado/core/html.py

```diff
@@ -298,7 +298,7 @@ def _render_report(self):
                         template)
             ui.critical("-" * 80)
             ui.critical("%s:\n\n", details)
-            ui.critical("%r\n\n", self.json)
+            ui.critical("%r\n\n", self.result_dict)
```

Comment:
IMHO it just makes a point: there was no JSON, it actually fixes that. And, it helps to understand the whole transition.